futurex_openedx_extensions/dashboard/urls.py

```diff
@@ -36,6 +36,9 @@
     re_path(
         r'^api/fx/learners/v1/enrollments/$',
         views.LearnersEnrollmentView.as_view(), name='learners-enrollements'),
+    re_path(
+        r'^api/fx/learners/v1/unenroll/$',
+        views.LearnerUnenrollView.as_view(), name='learner-unenroll'),
```

Comment:
fix the name, please
```suggestion
        views.LearnerUnenrollView.as_view(), name='learners-unenroll'),
```